examples/react/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "hotosm-ui-vue-example",
```

Comment:
Tiny thing - can we check the name of this please - in the react folder example! 😄